### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by Zemill** [Heroes-Profile/heroesprofile#703] [Ideas]
https://www.reddit.com/r/heroesofthestorm/comments/187ic6y/comment/kjoj88h/?context=3

**Comment by Zemill** [comment]
The infrastructure for this is set up.

Users set dark mode in profile settings.  might need styled a little

http://127.0.0.1:8000/Profile/Settings
![image](https://github.com/Heroes-Profile/heroesprofile/assets/33404722/63c32954-4021-407a-b475-09a6eb33ea17)


I made the value available in the layouts file
https://github.com/Heroes-Profile/heroesprofile/blob/60328daddf989102e1493dd673714989a860d047/resources/views/layouts/app.blade.php#L52

**Comment by lifeinabstract** [comment]
To add dark mode to an element, remove the background class and add .variable-background.
Also can use .variable-text; .variable-hover.

Used for anything with a light background.

**Comment by Zemill** [comment]
Implemented
